dvc/repo/pull.py

```diff
@@ -31,6 +32,9 @@ def pull(
 
     expanded_targets = glob_targets(targets, glob=glob)
```

Comment:
Could we raise the exception inside `glob_targets` instead?

Apart from pull and push, st is only being used in `dvc add` and I believe raising exception would also make sense there

Reply:
Thanks, done.